mjpc/tasks/humanoid/tracking/task.xml

```diff
@@ -0,0 +1,164 @@
+    <body name="mocap[ltoe]" mocap="true">
+      <site name="mocap[ltoe]" class="mocap_site" />
+    </body>
+    <body name="mocap[rtoe]" mocap="true">
+      <site name="mocap[rtoe]" class="mocap_site" />
+    </body>
+    <body name="mocap[lheel]" mocap="true">
+      <site name="mocap[lheel]" class="mocap_site" />
+    </body>
+    <body name="mocap[rheel]" mocap="true">
+      <site name="mocap[rheel]" class="mocap_site" />
+    </body>
+    <body name="mocap[lknee]" mocap="true">
+      <site name="mocap[lknee]" class="mocap_site" />
+    </body>
+    <body name="mocap[rknee]" mocap="true">
+      <site name="mocap[rknee]" class="mocap_site" />
+    </body>
+    <body name="mocap[lhand]" mocap="true">
+      <site name="mocap[lhand]" class="mocap_site" />
+    </body>
+    <body name="mocap[rhand]" mocap="true">
+      <site name="mocap[rhand]" class="mocap_site" />
+    </body>
+    <body name="mocap[lelbow]" mocap="true">
+      <site name="mocap[lelbow]" class="mocap_site" />
+    </body>
+    <body name="mocap[relbow]" mocap="true">
+      <site name="mocap[relbow]" class="mocap_site" />
+    </body>
+    <body name="mocap[lshoulder]" mocap="true">
+      <site name="mocap[lshoulder]" class="mocap_site" />
+    </body>
+    <body name="mocap[rshoulder]" mocap="true">
+      <site name="mocap[rshoulder]" class="mocap_site" />
+    </body>
+    <body name="mocap[lhip]" mocap="true">
+      <site name="mocap[lhip]" class="mocap_site" />
+    </body>
+    <body name="mocap[rhip]" mocap="true">
+      <site name="mocap[rhip]" class="mocap_site" />
+    </body>
+  </worldbody>
+
+  <sensor>
+    <user name="control" dim="21" user="3 0 0.0 0.1 0.3" />
+    <user name="joint vel." dim="27" user="0 0 0.0 0.1" />
+
+    <user name="root pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="head pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="ltoe pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="rtoe pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="lheel pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="rheel pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="lknee pos error" dim="3" user="6 10.0 0.0 100.0 0.1" />
+    <user name="rknee pos error" dim="3" user="6 10.0 0.0 100.0 0.1" />
+    <user name="lhand pos error" dim="3" user="6 60.0 0.0 100.0 0.1" />
+    <user name="rhand pos error" dim="3" user="6 60.0 0.0 100.0 0.1" />
+    <user name="lelbow pos error" dim="3" user="6 10.0 0.0 100.0 0.1" />
+    <user name="relbow pos error" dim="3" user="6 10.0 0.0 100.0 0.1" />
+    <user name="lshoulder pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="rshoulder pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="lhip pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="rhip pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+
+    <user name="root vel error" dim="3" user="6 0.003 0.0 1.0 0.1" />
```

Comment:
TODO: I'm not really sure if these are needed. The position errors might suffice.